meta-lmp-base/classes/lmp.bbclass

```diff
@@ -23,4 +23,19 @@ IMAGE_CMD_ota_append () {
 		${OTA_SYSROOT}/ostree/deploy/${OSTREE_OSNAME}/var/sota/import/installed_versions \
 			${GARAGE_TARGET_NAME}-${target_version}
 	fi
+
+    if [ "${DOCKER_COMPOSE_APP_PRELOAD}" = "1" ]; then
```

Comment:
Indentation is not aligned with the previous code block, using spaces instead of tabs. Mind changing to tab so it can be aligned?

Reply:
updated